### PR TITLE
[FEAT] Create API for POSTing preferences

### PR DESCRIPTION
**Pull request by Jaysce**
- Add ability to make a POST to the **/preferences** endpoint to create a preferences object in the Preferences collection in mongoDB

### CODE REVIEW

File: backend/src/routes/preferences.js [hunk 1]
```diff
@@ -1,6 +1,23 @@
 import express from 'express';
+import Preference from '../mongo/models/Preference';
 const router = express.Router();
 
-// TODO: Handlers...
+// POST: Creating a preferences document in the preferences collection
+router.post('/', async (req, res) => {
```
Comment (cecilia-yin) [inline backend/src/routes/preferences.js:6]: Your api contract says the path should be '/preferences'. Which one are you using?

Comment (Jaysce) [reply backend/src/routes/preferences.js:6]: The path is defined in index.js:
Screen Shot 2021-03-16 at 8 29 35 pm

Comment (cecilia-yin) [reply backend/src/routes/preferences.js:6]: oops my bad

**Review by blakehattingh — APPROVED** [review]
I assume further error handling will need to be implemented but this looks good for MVP.

**Review by lilichenyuxin — APPROVED** [review]
LGTM

**Pull request merged**